hardhat.config.ts

```diff
@@ -7,7 +7,7 @@ import "hardhat-abi-exporter"
 
 // deployment plugins
 import "hardhat-deploy"
-import "hardhat-deploy-ethers"
+import "@nomiclabs/hardhat-ethers"
```

Comment:
based on https://github.com/wighawag/hardhat-deploy-ethers#installation
the compile step was failing due to yarn.lock hence had to change this